tests/UKHO.ERPFacade.API.UnitTests/Controllers/ErpFacadeControllerTests.cs

```diff
@@ -53,15 +53,15 @@ public async Task WhenValidRequestReceived_ThenErpFacadeReturns200OkResponse()
             A.CallTo(_fakeLogger).Where(call => call.Method.Name == "Log"
              && call.GetArgument<LogLevel>(0) == LogLevel.Information
              && call.GetArgument<EventId>(1) == EventIds.BlobExistsInAzure.ToEventId()
-             && call.GetArgument<IEnumerable<KeyValuePair<string, object>>>(2)!.ToDictionary(c => c.Key, c => c.Value)["{OriginalFormat}"].ToString() == "Blob exists in the Azure Storage for the trace ID received from SAP event.").MustHaveHappenedOnceExactly();
+             && call.GetArgument<IEnumerable<KeyValuePair<string, object>>>(2)!.ToDictionary(c => c.Key, c => c.Value)["{OriginalFormat}"].ToString() == "Blob exists in the Azure Storage for the corrId received from SAP event.").MustHaveHappenedOnceExactly();
         }
 
         [Test]
-        public async Task WhenTraceIdIsMissingInRequest_ThenErpFacadeReturnsReturns400BadRequestResponse()
+        public async Task WhenCorrIdIsMissingInRequest_ThenPostPriceInformationReturnsReturns400BadRequestResponse()
```

Comment:
Can be renamed as WhenCorrIdIsMissingInRequest_ThenPostPriceInformationReturns400BadRequestResponse

Reply:
Renamed